tests/app_setup/test_db_setup.py

```diff
@@ -1,42 +1,58 @@
-from flask import current_app as app
-from flask_sqlalchemy import SQLAlchemy
-from sqlalchemy import create_engine
 import tests.factories as factories
 
 from app.serotracker_sqlalchemy import db_tables
-from tests.utils import delete_records
+from tests.utils import delete_records, delete_all_records, insert_full_recordset
 from app.serotracker_sqlalchemy.models import DashboardSource, ResearchSource, AntibodyTarget, AntibodyTargetBridge, \
-    City, CityBridge, State, StateBridge, TestManufacturer, TestManufacturerBridge
+    City, CityBridge, State, StateBridge, TestManufacturer, TestManufacturerBridge, Country
 
 
-def test_attached_db(db):
-    assert isinstance(db, SQLAlchemy)
-
-
-def test_db_table_names():
-    engine = create_engine(app.config['SQLALCHEMY_DATABASE_URI'])
+def test_db_table_names(db):
     for table in db_tables:
-        assert table in engine.table_names()
+        assert table in db.engine.table_names()
```

Comment:
Don't we also need to test this the other way?

You could assert that the two table names are identical // equivalent sets, or otherwise assert that every table in db.engine.table_names() is in db_tables to test it the other way

Reply:
Yeah we could do that. Also my suspicion is that there should be a way to infer all tables from a models file so we shouldn't need db_tables anyways